### ISSUE TITLE
User is not able to activate the 'Drop Down' in NVDA browse mode using 'Space/Enter' key.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13300]
### Steps to reproduce:
1.Open above URL : https://a830edad9050849387e18050220.sharepoint.com/_layouts/15/sharepoint.aspx? in Edge Anaheim.
2.Create a comm site and turn on the screen reader NVDA
3. Navigate to any 'Drop Down' control in header section using Down arrow key in NVDA browse mode.
4. Try to activate 'Drop Down' using 'Space/Enter' key in NVDA browse mode.
5. Verify that user is able to activate 'Drop Down' control or not.

### Actual behavior:
User is not able to activate the 'Drop Down' in NVDA browse mode using 'Space/Enter' key.
While user can activate 'Drop Down' control using 'Alt + Down' arrow key in NVDA scan mode.

### Expected behavior:
User should be able to activate the 'Drop Down' control using 'Space/Enter' key in NVDA browse mode.
### System configuration
Browser Version: 93.0.933.1 (Official build) dev (64-bit)
OS Version: 21H1 (OS Build 19043.1052)
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA Version 2021.3
#### Windows version:
OS build Version 20H2 (OS Build 19042.508)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, NVDA Version 2021.3.1
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

https://user-images.githubusercontent.com/93735775/152242654-c6d7d7b8-f283-4a15-9045-401492bb5387.mp4

**Comment by msftedad** [comment]
Hi. could you please give us an update on this issue which was logged in February month? Can you give us an ETA for fixing this bug?

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Closing as still blocked by sample
